### PR TITLE
add information about this repository to README file

### PR DESCRIPTION
**Pull request by igrr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,17 @@
...
+
+The releases will be updated whenever a new Espressif chip or a chip revision is released.
+
+## Copyrights and License
+
+The ROMs of ESP32 series of chips are Copyright (c) 2015-2022 Espressif Systems (Shanghai) Co. Ltd.
+
+The ROMs also include various third-party libraries. The list along with the respective licenses is available [here](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/COPYRIGHT.html#rom-source-code-copyrights).
+
+The ROM code of the currently available ESP32 series of chips is not open-source.
```
Comment (Lapshin) [inline README.md:13]:
```suggestion
The ROM source code of ESP32 series chips is proprietary and confidential.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,17 @@
...
+
+You can use the ELF files in this repository under the terms of Apache License 2.0, provided in the file [LICENSE](LICENSE).
+
+Despite the fact that Apache license is usually applied to software in the source form, the license is also applicable to the compiled files — they are referred to in the license text as _"Object" form_. We chose to release the ROM ELF files under this license because that makes it easier to use these ELF files in the context of other Apache licensed projects.
```
Comment (Lapshin) [inline README.md:17]:
```suggestion
Despite the fact that Apache license is usually applied to software in the source form, the license is also applicable to the compiled files — they are referred to the license text as _"Object" form_. We chose to release the ROM ELF files under this license because that makes it easier to use these ELF files in the context of other Apache licensed projects.
```

Comment (igrr) [reply README.md:17]: I think that 'in' was intentional... another way to put this is, "In the license text, they are referred to as 'Object form'". I'll rephrase this to make it sound less odd.

File: README.md (hunk 1 above)
Comment (igrr) [reply README.md:13]: While I think the way you put it is technically correct, I think there is a slight difference in the perceived meaning :)

I wanted to say that just for the chips available now, the ROM is not open source. (Implied: we might be able to make it open source for the future chips.)

On the other hand, "the ROM code is proprietary and confidential" sounds like a stubborn position of a chip vendor who has decided that it's the way this will be, forever.

File: README.md [hunk 3]
```diff
@@ -1 +1,17 @@
-# esp-rom-elfs
+# ESP32 series ROM ELF files
+
+The [Releases](https://github.com/espressif/esp-rom-elfs/releases) page of this repository is used to publish the ELF files of ROMs used by ESP32 series of chips.
+
+The releases will be updated whenever a new Espressif chip or a chip revision is released.
+
```
Comment (Lapshin) [inline README.md:6]: I'm editing IDF's docs and think we need to say a few words about how to choose the right ROM ELF revision:

```suggestion
## Choosing ROM ELF for a chip

In the release archives, you can find files with two filename patterns:

1. <CHIP_NAME>_rom.elf
2. <CHIP_NAME>_rev<REV>_rom.elf

If you found a ROM ELF with the first pattern, just use it.
Otherwise, find out your chip revision and use ROM ELF file with <REV> equals chip's revision. Or if the file does not exist use the last <REV> lower than the chip's revision. E.g.: for `esp32 rev2` use  `esp32_rev0_rom.elf`.
```

File: README.md (hunk 1 above)
Comment (igrr) [reply README.md:23]: After thinking about this for a bit, I have opted to keep it as is. Thanks for the suggestions though!

File: README.md [hunk 4]
```diff
@@ -1 +1,27 @@
...
+## Choosing the ROM ELF file
+
+In the release archives, you can find files with two filename patterns:
+
+1. <CHIP_NAME>_rom.elf — for chips where the ROM code is the same in all chip revisions.
+2. <CHIP_NAME>_rev<REV>_rom.elf — for chips where the ROM code differs between chip revisions. You need to use the version of ROM with the revision equal to the revision of your chip. If such file is not available, use the next lower one.
```
Comment (igrr) [inline README.md:12]: @Lapshin what do you think about _always_ adding `_rev0_` for the chips which only have one ROM revision? The benefit would be that there is no "special case" for handling such chips. Also if the new revision is added, we wouldn't have to modify the existing file name, will just add a new one.

Comment (Lapshin) [reply README.md:12]: @igrr , yes, it would be more convenient for all. Let me do this.

Comment (igrr) [reply README.md]: @Lapshin I have updated this part, PTAL again.

Comment (Lapshin) [reply README.md]: LGTM! Thank you, @igrr
